hazelcast/src/main/java/com/hazelcast/instance/Node.java

```diff
@@ -420,7 +420,8 @@ public void shutdown(final boolean terminate) {
         }
     }
 
-    public void onRestart() {
+    public void reset() {
```

Comment:
Can you add javadoc what the purpose is of 'reset'. Else it will not be maintainable by other engineers.